tests/acceptance/features/apiArchiver/downloadByPath.feature

```diff
@@ -65,12 +65,12 @@ Feature: download multiple resources bundled into an archive
       | my_data/textfile2.txt       | some data  |
       | more_data/an_other_file.txt | more data  |
 
-
+  
```

Comment:
```suggestion

```

Reply:
I change it in next PR. I don't want to run CI again